### PR TITLE
Additional grammar fixes ( have some doubts)

### PR DESCRIPTION
**Pull request by LShun**
Hi, changed some more and left notes in the commits, please help me to check them and correct me if there's any mistakes. I also used some based on Russian translation, thanks!

### CODE REVIEW

File: Translate/En/21-08.md [hunk 1]
```diff
@@ -1,118 +1,121 @@
 ## Part 8.
 
-Red armored boots trampled on the right hand of Asuna stretched towards Kirito.
+A red armored boot trampled on top of Asuna's right hand, which was stretched towards Kirito.
 
-As I looked up, a red knight, who caused both eyes behind the helmet to shake the horrible hatred, kept sweating the long sword backed up with both hands.
+As I looked up, a red knight, whose both eyes filled with horrible hatred caused the helmet to seemingly shake, started gripping the long sword with both hands.
```
Comment (AgentMC) [inline Translate/En/21-08.md]: To accommodate differences between the sentence construction of English and Japanese, Google Translate adds pronouns (and yes, original text doesn't have them). AI "guesses" but obviously it can only guess within a sentence, so beyond that it usually chooses 1st-person (I). The "I" in this book can actually be used only if
- we're talking of Kirito, whose POV is traditionally given 1st-person,
- we're in thoughts or a quote of a character.

Basically, to summarise, here and in similar places it should be "she".
BTW in this case I think his eyes behind the helmet's visor were trembling from the hatred, not the helmet shaking :)

File: Translate/En/21-08.md [hunk 2]
```diff
@@ -1,118 +1,121 @@
...
 
-A sword is thrust down, with a stinging screech.
+A sword is thrust down with a stinging screech.
 
-There is no power to fight anymore, but Asuna stares at the steel iron trying to penetrate her, at least not whether the eyes are closed.
+There is no power within her to fight anymore, but Asuna still stares at the steel blade as it attempts to pierce her, at least her eyes are not closed(??).
```
Comment (AgentMC) [inline Translate/En/21-08.md]: She has no physical power left, but at least her eyes are still open, so she watches the blade approaching her. E.g. "I'll die but with my eyes open".

File: Translate/En/21-08.md [hunk 3]
```diff
@@ -1,118 +1,121 @@
...
+"Wow...?"
 
-A knight who revealed a disconcerting voice will swing down the sword again. But the sparks scatter again, the sword does not reach Asuna.
+The knight who revealed a disconcerting voice swings down the sword again. However, sparks scatters again, the sword is unable to reach Asuna.
 
-For the third time, the fourth time, the result is the same.
+Even for the third time, the fourth time, the results are still the same.
 
-The fifth attack was not done. Solterina, who came running to Asuna, played the Red Knight greatly with the sword skill "Torrent" for knockback using the handle of the large sword. While helping Asuna, Solterina asked surprise in a hidden voice.
+Before the fifth attack was executed, Solterina, who came running to Asuna, pushed(?) the Red Knight back(?) greatly with the sword skill "Torrent", producing knockback using the handle of the large sword. While helping Asuna up, Solterina asked surprisingly in a small voice.
```
Comment (AgentMC) [inline Translate/En/21-08.md]: 👍

File: Translate/En/21-08.md [hunk 4]
```diff
@@ -1,118 +1,121 @@
...
 
-"The "sword of mind" now ... ... Asuna !?"
+"Asuna...was that the "Sword of Mind" just now...!?"
 
-"Incarnation ... ...?"
+"What's the incarnation...?"
```
Comment (AgentMC) [inline Translate/En/21-08.md]: OK... let's think logically...
Asuna knows the incarnational techniques, because she's the f..ng Stacia capable of adjusting the terrain. So my bet would be a 'ken' got lost here and instead of Shin-Ken (sword of mind)  I've got "Shin-...", guessed "ii", translated as incarnation and left as is.
She should here repeat after Solterina the unknown term;
`"A 'Sword of Mind'..?"`

...Yeah, and next line is also 'she', of course.

File: Translate/En/21-08.md [hunk 5]
```diff
@@ -1,118 +1,121 @@
...
-Shake head a little to the word you listen for the first time.
+I shoke my head a little towards the word I heard for the first time.
 
 "No, it is not me."
 
-"Well ...... Renri is ... ...."
+"Well, Renri is..."
 
-As Asuna chased the line of sight of Solterina turning around while saying that, the boy knight who was seriously injured was flying instructions to the troops to intercept the army of the approaching red knight, and he said that he was able to afford to keep Asuna very much I do not think so.
+As Asuna followed Solterina's line of sight as she turns around while saying that, the male knight, who was terribly injured was giving instructions to the troops to intercept the army of the approaching red knight, said he don't think he is able to afford to keep an eye on Asuna any much longer. (??)
```
Comment (AgentMC) [inline Translate/En/21-08.md]: ...and by the look of it Asuna ensures sure that Renri has no time/power left for shooting protective incarnations around.

File: Translate/En/21-08.md [hunk 6]
```diff
@@ -1,118 +1,121 @@
...
+However, while we are trying to solve this phenomenon, we must first defend as many Underworld people as possible.
 
-Asuna stood up with the help of Solterina, inspiring the exhausted spirit and confirmed the circumstances.
+Asuna stood up with the help of Solterina, inspiring the exhausted spirit, as she assess the circumstances.
 
-And I felt something new despair came into my mind as it turned black and cold water.
+And I felt something new, and full of despair came into my mind, like cold dark water pouring in.
 
-More than 80% of players in China and Korea over ten thousand have already entered battle between ally. Even if it is said, the morale difference, the main faction is overwhelming the other players. Avatar disappearing here and there Blue light pillar of effect rises, every time a ferocious scream echoes.
+More than 80% of players in China and Korea, over ten thousand players have already entered the battle between allies. Even if it is said, the morale difference, the main faction is still overwhelimg the resisting factions. As Avatar's disappear here and there, effects in a pillar of blue light rises, and every time, a ferocious scream echoes.
```
Comment (AgentMC) [inline Translate/En/21-08.md]: 'avatars' then (lowercase)

File: Translate/En/21-08.md [hunk 7]
```diff
@@ -1,118 +1,121 @@
...
 
-In addition, some of the main factions ……There is little battle capability left for the Japanese, and the underworld people including the integrity knight Renri also are full of creation. Even if it says that there is an advantage of sacred art and sword skill, I do not think it can be repulsed very much.
+In addition, some of the moderate factions…… There is little battle capability left in the Japanese, and the underworld people including the integrity knight Renri also which are all on their last breath(??). Even if there is an advantage from the possession of sacred art and sword skills, I do not think the opposition can be repulsed much.
```
Comment (AgentMC) [inline Translate/En/21-08.md]: "Creation" = "Destiny" = "_life_ of character". Should be smth like
In addition, not only the moderate/resisting _(choose one)_ faction...little potential to fight is _also_ left in [...] which are _running out of life_.`

File: Translate/En/21-08.md [hunk 8]
```diff
@@ -1,118 +1,121 @@
...
 
-I can not find any words anymore, so I can only sit in the arm of Solterina.
+I cannot find any words anymore, I can only stand up while gripping Solterina's arm.
 
-With several echoes Loud laughter of the POH has arrived..
+Echoes of loud laughter from PoH arrives...
 
-Beside the lying Kirito, the death god who left a large hole in his chest raised his right hand and a five-finger extended left hand holding the giant friendship kitchen knife at the highest and laughed with his upper body rebelled against him . One day the black clouds are huge swirls in the sky, resources of life which is dissipated in the battlefield are hung down on the tornado and are sucked into the body of POH.
+Beside Kirito, who was lying down, the death god who has a large hole in his chest raised his right hand and all his five-finger extended and his left hand, which is holding the giant friendship kitchen knife at its highest and laughed with his upper body resisting its weight. Huge black clouds swirls in the sky at daytime, a funnel of life resources dissipated in the battlefield forms a tornado as they all are absorbed into PoH's body.
```
Comment (AgentMC) [inline Translate/En/21-08.md]: BTW, Defan had a proper name for that knife, something like 'Knife for your friend' but not sure.

File: Translate/En/21-08.md [hunk 9]
```diff
@@ -1,118 +1,121 @@
...
 
-To be precise, it is the right magic sword that absorbs resources. If we can destroy that, the energy supply to the owner will be cut off, the death god who lost the heart should die immediately.
+To be precise, it is the right magic sword that absorbs life resources. If we can destroy that, the energy supply to the owner will be cut off, the death god who lost his heart should die immediately.
```
Comment (AgentMC) [inline Translate/En/21-08.md]: ...magic sword in his right hand...

File: Translate/En/21-08.md [hunk 10]
```diff
@@ -1,118 +1,121 @@
...
 
-However, the situation has already come to the point where even if one of the enemy commanders is defeated it can not be scooped. The mainist factions driven by PoH's inciting words and evil talent, even if the commander leaves, continues fighting until it kills Japanese and underworld people as a source of further anger I will. In the case of
+However, the situation has already come to the point where even if one of the enemy commanders is defeated it can no longer be salvaged. The main factions is driven by PoH's inciting words and evil talent. Even if the commander leaves, they will continue fighting until they kill all the Japanese and Underworld people who are also acting as a source of further anger.
 
-_How should I act? What to do?_
+In the case,
```
Comment (AgentMC) [inline Translate/En/21-08.md]: Some jerk trained Google translate to "translate" isolated space ( ) to a phrase of "In the case of". Funny thing that AI picked it and now I'm seeing an interesting behavior of how it tries to concatenate it to real sentences... anyway, please remove that portion entirely.

File: Translate/En/21-08.md [hunk 11]
```diff
@@ -1,118 +1,121 @@
...
 
-Asuna aroused by frustration and despair, she noticed a strange phenomenon.
+_How should I act? What should I do?_
 
-A white fog is drawing thin on the ground where dark gravel has been drawn until a while ago. While sliding like ribbons made of thin silk, they pass through the legs of Asuna and spread backwards. At the same time, a refreshing sweet scent tickles the nostrils.
+As Asuna was being torn by frustration and despair, she noticed a strange phenomenon.
+Oon the ground where dark gravel has been drawn until a while ago, white fog is spreading thinly on top of it. Sliding like ribbons made of thin silk, they under Asuna's as it continues to spread backwards. At the same time, a refreshing sweet scent starts to tickle her nostrils.
```
Comment (AgentMC) [inline Translate/En/21-08.md]: "On", I guess + "they under Asuna's"??? :D

File: Translate/En/21-08.md [hunk 12]
```diff
@@ -1,118 +1,121 @@
...
+To be precise, it is the long blue sword, grasped with his left hand. The blade is broken from the middle, but the mist wraps its whole, it seems as if it emits light faintly.
 
-"Kirito-kun"
+"Kirito-kun."
 
-At the same time Asuna calls the name of the person who loves more than anyone in the shivering voice.
+At the same time Asuna calls the name of the person who she loves more than anyone in a shivering voice.
 
-In the case of
+In the case,
```
Comment (AgentMC) [inline Translate/En/21-08.md]: Yet again a rogue space in the source...

File: Translate/En/21-08.md [hunk 13]
```diff
@@ -1,118 +1,121 @@
...
 
-"That “sword of mind” ... ... Kirito’s ..."
+"That “sword of mind” just now, is Kirito’s..."
```
Comment (AgentMC) [inline Translate/En/21-08.md]: "was" or "has been", probably

File: Translate/En/21-08.md [hunk 14]
```diff
@@ -1,118 +1,121 @@
...
 
-The white fog reaches the feet of the Allied Chinese-Korean Army that surrounds the periphery, and it continues to spread still. Although they do not seem to be noticing in battle, they are already being swallowed by pure white ribbons around the knees. At last, it seemed that POH who kept on laughing also noticed something strange.
+The white fog reaches the feet of the Allied Chinese-Korean Army that surrounds the peripherial space, and it still continues to spread. Although they do not seem to notice it because as they are in the midst battle, they are already being swallowed by pure white ribbons around the knees. At last, it seemed that POH who kept on laughing has also noticed something strange.
 
-First of all, I stare at the footsteps and then see Kirito behind like it was played. Do not make a tall person strong for a moment, if you rotate a friend knife kitchen knife on the right and grip it with a rush, it gets to a great deal of trouble.
+First of all, he lowered his gaze towards the footsteps, and then, he saw Kirito behind him. Wasting no time, he threw his "Friendship Cleaver" in his right hand and, but it was easily intercepted.
```
Comment (AgentMC) [inline Translate/En/21-08.md]: PoH standing in ~5 steps from Kirito, so he raises the "knife", and 
"...should he run and hit, the damage would be extreme..."
There is no hit done yet.

**Review by AgentMC — COMMENTED** [review]
I decided not to directly interfere but to leave comments to show you more GT tricks and hopefully make it easier for future updates.
P.S.: yeah, this all should be past tense (as I explain in the comment on "I" but that would be too much work really)

File: Translate/En/21-08.md (hunk 1 above)
Comment (LShun) [reply Translate/En/21-08.md]: I agree, was previously wondering why is the helmet shaking from hatred but dismissed it as clenching teeth and such, thanks!

File: Translate/En/21-08.md (hunk 2 above)
Comment (LShun) [reply Translate/En/21-08.md]: Alright, I changed this slightly in the next translation but I'm still doubting it, need some confirmation.

File: Translate/En/21-08.md (hunk 7 above)
Comment (LShun) [reply Translate/En/21-08.md]: I'm thinking of changing it to "there isn't any much potential to fight left in" instead, how do you think?

File: Translate/En/21-08.md (hunk 11 above)
Comment (LShun) [reply Translate/En/21-08.md]: lmao, thanks, was thinking of they spread under her feet, cause the next part is about the Enhance Armament, if it actually goes on her legs she will be frozen too, I guess?

**Review by AgentMC — APPROVED** [review]
Great work, thanks.

**Comment by AgentMC** [comment]
@LShun I only worry a bit regarding "avatar's were" in couple of places. Guess it should be "avatars were". But anyway it can go as a separate change.
